.github/workflows/testAndPublish.yml

```diff
@@ -28,7 +28,9 @@ env:
   START_BUILD_NUMBER: 50000
   pullRequestNumber: ${{ github.event_name == 'pull_request' && github.event.number || 0 }}
   scons_publisher: NV Access
+  # Don't send telemetry to Microsoft when using MSVC tooling, avoiding an unnecessary PowerShell script invocation.
   VSCMD_SKIP_SENDTELEMETRY: 1
+  # Cache details about available MSVC tooling for subsequent SCons invocations to the provided file.
   SCONS_CACHE_MSVC_CONFIG: ".scons_msvc_cache.json"
```

Comment:
does this need to be here? or can it just be in the cache action?

Reply:
It needs to be there for every invocation of SCons.